falcon_multiqc/commands/sql.py

```diff
@@ -0,0 +1,52 @@
+import click
+import glob
+import os
+from sqlalchemy import create_engine
+from database.config import DATABASE_URI
+from database.process_query import run_multiqc
+
+# example query: 
+
+# this query will query fastqc tool for Total seqence metric - since only the first batch ran fastqc it will only return first batch
+# SELECT s.sample_name, b.path FROM public.raw_data as rd join sample as s on (s.id = rd.sample_id) join batch as b on (s.batch_id = b.id) where rd.qc_tool = 'fastqc' AND (rd.metrics->>'Total Sequences')::numeric::integer >= 316624794;
+
+# this queries two different tools (picard_insertsize and verifybamid) and on of their respective metrics - returns sampleID from first batch and sampleID from 4012 batch 
+# SELECT s.sample_name, b.path FROM public.raw_data as rd join sample as s on (s.id = rd.sample_id) join batch as b on (s.batch_id = b.id) where ((rd.qc_tool = 'picard_insertSize' AND (rd.metrics->>'MEAN_INSERT_SIZE')::numeric::integer <= 390) OR (rd.qc_tool = 'verifybamid' AND (rd.metrics->>'#READS')::numeric::integer >= 4850000));
+
+@click.command()
+@click.option("-d", "--directory", type=click.STRING, required=True, help="where query result will be saved")
```

Comment:
Right now this is going to overwrite my query output each time I run this again in the same directory. Do you think maybe we should set it up so this is instead a path to output (with the filename output)? So instead this command just supports 1 query at a time. Not 100% on this, what do you think?

Reply:
Possibly, but actually I've added the 'count' variable which slightly changes the name of the query result file each time, so you can run it as many times as you want in the same directory and it will just keep adding - multiqc also automatically adds (_1 _2, _3) to the end of their newly generated files in the same directory

Reply:
Yeah that's cool, I just meant if I run the program again it's going to start at 0 again and overwrite the old ones.

To be honest, I think it's just best to keep commands very straight forward, take in arguments, spit out the result, instead of waiting for user input. This would be easier for them since it's probably being sent to a cluster and they won't be able to put in that user input.

Reply:
yeah true actually, alrighty